docs/examples/backends/backends_example.ipynb

```diff
@@ -105,9 +105,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "aer_state_b = AerStateBackend()\n",
-    "state_handle = aer_state_b.process_circuit(circ)\n",
-    "statevector = aer_state_b.get_result(state_handle).get_state()\n",
+    "are_state_b = AerStateBackend()\n",
+    "state_handle = are_state_b.process_circuit(circ)\n",
+    "statevector = are_state_b.get_result(state_handle).get_state()\n",
```

Comment:
This should have probably stayed as `aer_state_b`

Reply:
Ah good catch, will fix.